src/main/java/org/opentripplanner/middleware/docs/ReadMeEnvSchemaValuesUpdater.java

```diff
@@ -69,7 +69,9 @@ public static String generateEnvSchemaValuesContent() throws IOException {
                 .append(" | ")
                 .append(required)
                 .append(" | ")
-                .append(defaultValue.replaceAll("\"", ""))
+                .append(examples.replaceAll("\"", "")
```

Comment:
If `examples` was an array or list, you could simply write `Strings.join(", ", examples)`.